sphinx/users/comlinetools/precompressed.rst

```diff
@@ -0,0 +1,31 @@
+"Precompressed" tiles
+=====================
+
+See first :doc:`a summary of the precompressed tiles feature </formats/precompressed>`.
```

Comment:
Interestingly the `conversion.rst` page also mentions a `Summary of the precompressed tiles feature` but links to this page. Should this be `a summary of the formats supporting precompressed tiles features` instead?
Or alternatively is this not necessary as the link below links directly to the readers/writers section?

Reply:
My original thought here was that `formats/precompressed.rst` is the higher-level motivation for why you might want to use precompressed tiles, where `users/comlinetools/precompressed.rst` and `users/comlinetools/conversion.rst` are talking about the specific implementation in `bfconvert`. So if you didn't already know why this feature might be useful, you'd want to read at least the first section of `formats/precompressed.rst` to understand.

4d06960 tries to clarify the link text a bit, but open to other suggestions.